clusterloader2/pkg/measurement/util/util.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Do you really need to create another 'util' directory for this file? Maybe move it to measurement/ ?

Reply:
I've moved it to separate directory because this functionality is not necessary for measurement.
It's very convenient to have this kind of util, however measurement can be implemented without it.

Reply:
If it's not a measurement specific util, I'd suggest moving it outside. Probably into pkg/util.

Reply:
ok. done.